atomic_reactor/plugins/pre_reactor_config.py

```diff
@@ -131,6 +131,27 @@ def get_smtp_session(workflow, fallback):
     return smtplib.SMTP(config['host'])
 
 
+def get_cachito(workflow):
+    return get_value(workflow, 'cachito', NO_FALLBACK)
+
+
+def get_cachito_session(workflow):
+    config = get_cachito(workflow)
+    from atomic_reactor.cachito_util import CachitoAPI
```

Comment:
Why is this import here in function? Cyclic imports?

Reply:
I followed the pattern in `get_odcs_session`. IIRC, this is so we can split up the plugin into subpackages. If that's not needed, I can move it to the top.

Reply:
Please put it on top of the module

Reply:
Done!